pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -117,6 +119,10 @@ type VaultAppRole struct {
 	SecretRef SecretKeySelector `json:"secretRef"`
 }
 
+type KubernetesAuth struct {
+	Role string `json:"role"`
```

Comment:
What's the purpose of the `role` field?

All new API fields should also have comments to explain them 😄